addons/mail/static/src/models/chat_room/chat_room.js

```diff
@@ -0,0 +1,119 @@
+odoo.define('mail/static/src/models/chat_room/chat_room.js', function (require) {
```

Comment:
For clarity, would use naming "RTC room". That's already the name of the python model, and I like it the most.